CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed
 
+- Allow configuring the inline ratio and auto-resizing for manually defined
+  LTIConsumerPlugin isntances
```

Comment:
```suggestion
- Allow configuring the inline ratio and auto-resizing manually for defined
  LTIConsumerPlugin instances
```

Reply:
actually this would mean something different. I really meant "for manually-defined instances". Let me add the "-".

Reply:
Oh ok!